sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_custom_forms_async.py

```diff
@@ -71,6 +71,16 @@ async def recognize_custom_forms(self):
                     print("...Label '{}' has value '{}' with a confidence score of {}".format(
                         field.label_data.text if field.label_data else name, field.value, field.confidence
                     ))
+                # [END recognize_custom_forms_async]
+
+                # iterate over tables on each page
+                for page in form.pages:
```

Comment:
hmm why only tables? should we show selection mark, lines, etc? or not include it at all

Reply:
docs team's feedback was specifically on highlighting tables. We already have a [sample ](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_get_bounding_boxes.py) that goes into the lines, selection marks, words elements in more details.

Reply:
But if someone only looks here, might think that the service only returns tables in the form type... so idk... maybe a comment after this that says there is also more information on pages? or at the beginning say some of the things that user can find in pages, and for example, here is the automatically recognized table: ...

Reply:
I think I was hoping to keep this sample more focused initially. But now that we are including tables here, I guess it would make sense to include lines and selection marks as well. will update